azurerm/provider.go

```diff
@@ -108,118 +108,119 @@ func Provider() terraform.ResourceProvider {
-			"azurerm_role_definition":                     resourceArmRoleDefinition(),
-			"azurerm_route":                               resourceArmRoute(),
-			"azurerm_route_table":                         resourceArmRouteTable(),
-			"azurerm_search_service":                      resourceArmSearchService(),
-			"azurerm_servicebus_namespace":                resourceArmServiceBusNamespace(),
-			"azurerm_servicebus_queue":                    resourceArmServiceBusQueue(),
-			"azurerm_servicebus_subscription":             resourceArmServiceBusSubscription(),
-			"azurerm_servicebus_subscription_rule":        resourceArmServiceBusSubscriptionRule(),
-			"azurerm_servicebus_topic":                    resourceArmServiceBusTopic(),
-			"azurerm_servicebus_topic_authorization_rule": resourceArmServiceBusTopicAuthorizationRule(),
-			"azurerm_snapshot":                            resourceArmSnapshot(),
-			"azurerm_scheduler_job_collection":            resourceArmSchedulerJobCollection(),
-			"azurerm_sql_database":                        resourceArmSqlDatabase(),
-			"azurerm_sql_elasticpool":                     resourceArmSqlElasticPool(),
-			"azurerm_sql_firewall_rule":                   resourceArmSqlFirewallRule(),
-			"azurerm_sql_active_directory_administrator":  resourceArmSqlAdministrator(),
-			"azurerm_sql_server":                          resourceArmSqlServer(),
-			"azurerm_sql_virtual_network_rule":            resourceArmSqlVirtualNetworkRule(),
-			"azurerm_storage_account":                     resourceArmStorageAccount(),
-			"azurerm_storage_blob":                        resourceArmStorageBlob(),
-			"azurerm_storage_container":                   resourceArmStorageContainer(),
-			"azurerm_storage_share":                       resourceArmStorageShare(),
-			"azurerm_storage_queue":                       resourceArmStorageQueue(),
-			"azurerm_storage_table":                       resourceArmStorageTable(),
-			"azurerm_subnet":                              resourceArmSubnet(),
-			"azurerm_template_deployment":                 resourceArmTemplateDeployment(),
-			"azurerm_traffic_manager_endpoint":            resourceArmTrafficManagerEndpoint(),
-			"azurerm_traffic_manager_profile":             resourceArmTrafficManagerProfile(),
-			"azurerm_virtual_machine_extension":           resourceArmVirtualMachineExtensions(),
-			"azurerm_virtual_machine":                     resourceArmVirtualMachine(),
-			"azurerm_virtual_machine_scale_set":           resourceArmVirtualMachineScaleSet(),
-			"azurerm_virtual_network":                     resourceArmVirtualNetwork(),
-			"azurerm_virtual_network_gateway":             resourceArmVirtualNetworkGateway(),
-			"azurerm_virtual_network_gateway_connection":  resourceArmVirtualNetworkGatewayConnection(),
-			"azurerm_virtual_network_peering":             resourceArmVirtualNetworkPeering(),
+			"azurerm_application_gateway":                  resourceArmApplicationGateway(),
```

Comment:
why did you change so many lines?

Reply:
Because one space is added to align with each other.

Reply:
on that note: you can use `?w=1` to ignore whitespace changes in reviews in Github, it's really handy for things like this :)